### PR TITLE
AV-77311 Capella landing pages

### PR DESCRIPTION
**Pull request by osfameron**
(see paired PR for docs-capella https://github.com/couchbase/docs-capella/pull/31 )

### CODE REVIEW

File: src/css/landing-page.css [hunk 1]
```diff
@@ -268,36 +268,6 @@
...
-.doc.landing-page-capella .top-row > .top-row-text-container {
-  flex-direction: column;
-  padding-right: 1rem;
-  position: relative;
-}
-
-.doc.landing-page-capella .top-row > .top-row-text-container > h1 {
-  font-weight: 600;
-  margin-top: 1rem;
-}
```
Comment (sarahlwelton) [inline src/css/landing-page.css]: I had all of this in here to control the positioning of the stars and the text in relation. How are we going to display that image with none of these controls?

File: src/partials/body-landing-page-capella.hbs [hunk 2]
```diff
@@ -13,6 +13,9 @@
 <main class="capella-article" data-ceiling="topbar">
 <article class="doc landing-page-capella">
 {{{page.contents}}}
+<div class="ending-image">
+    <img src="./_images/lineplus.svg" />
+</div>
```
Comment (sarahlwelton) [inline src/partials/body-landing-page-capella.hbs]: This image is not displaying at all on the Capella intro page by adding it in here. We need to troubleshoot.

**Review by sarahlwelton — CHANGES_REQUESTED** [review]
Need to know the way forward, because, again, not content to merge as-is.

File: src/partials/body-landing-page-capella.hbs (hunk 2 above)
Comment (osfameron) [reply src/partials/body-landing-page-capella.hbs]: Ah ok, it's a path issue.
We can move it into the UI root, and then it won't have that problem.

In the meantime it displays on https://preview.docs-test.couchbase.com/hakim-capella-landing/cloud/index.html 
(needs some minor CSS tweaking outdent it slightly)

File: src/css/landing-page.css (hunk 1 above)
Comment (osfameron) [reply src/css/landing-page.css]: Yeah, so arguably the Asciidoc model doesn't give us the flexibility here to set *part* of a direct hierarchy into another container just for formatting reasons.

I don't think it's worth the HTML and complexity - it belongs in the UI really in any case, I'll see if I can turn it into a background item.

**Pull request merged**